File: j-LIB-core/src/main/java/io/github/dot166/jLib/ThemeEngine/values.java
package io.github.dot166.jLib.ThemeEngine;

import android.content.Context;

import androidx.compose.material3.Shapes;
import androidx.compose.material3.Typography;
import androidx.compose.runtime.Composable;

public interface values {

    public int jLibTheme();

    public int M3();

    @Composable
    public Typography composeTypography(Context context);

    @Composable
    public Shapes composeShapes(Context context);
}

File: j-LIB-core/src/main/java/io/github/dot166/jlib/themeengine/values.kt
package io.github.dot166.jlib.themeengine

import android.content.Context
import androidx.compose.material3.Shapes
import androidx.compose.material3.Typography
import androidx.compose.runtime.Composable

@Deprecated("ThemeEngine is no longer maintained", level = DeprecationLevel.WARNING)
interface values {
    fun jLibTheme(): Int

    fun jLibClassicTheme(): Int

    fun M3(): Int

    @Composable
    fun composeTypography(context: Context?): Typography?

    @Composable
    fun composeShapes(context: Context?): Shapes?
}
